DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/_CentreCourseCard.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@using DigitalLearningSolutions.Data.Models.Courses
+@model CourseStatistics
+
+<div class="searchable-element nhsuk-panel card-with-buttons" id="@Model.CustomisationId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.CustomisationId-name" name="name">
+        @Model.CourseName
+      </span>
+    </summary>
+    <div class="nhsuk-details__text">
+
+    </div>
+  </details>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full nhsuk-u-margin-left-4">
+      <a class="nhsuk-button nhsuk-u-margin-right-2" role="button"
+         asp-controller="LearningMenu" asp-action="Index" asp-route-customisationId="@Model.CustomisationId">
+        Launch course
+      </a>
+      <a class="nhsuk-button nhsuk-button--secondary " role="button" asp-controller="TopCourses" asp-action="Index">
+        Manage course
+      </a>

```

Comment:
One thing I've noticed when we have two buttons/links next to each other on the same row is that NVDA seems to read them together in one quick burst, and the second button cannot be highlighted/stopped on when just using the arrow keys. Tabbing works fine. I don't know if this is just specific to my set up of NVDA, but there is a setting in Browse Mode that fixes this that I've enabled.